src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -120,62 +157,428 @@ def _load_iges(self):
             )
         return Iges(self._mapdl._generate_iges())
 
-    def _reset_cache(self):
+    def _reset_cache(self) -> None:
         self._keypoints_cache = None
         self._lines_cache = None
 
+    def __setitem__(self, key: Any, value: Any):
+        raise NotImplementedError("This method has not been implemented yet")
+
+    def __getitem__(self, name: str):
+        name = name.lower()
+        if "kp" in name:
+            return self.keypoints[name]
+        elif "line" in name:
+            return self.lines[name]
+        elif "area" in name:
+            return self.areas[name]
+        elif "volume" in name:
+            return self.volumes[name]
+        else:
+            raise ValueError(
+                f"The entity named '{name}' does not exist as a geometry entity."
```

Comment:
Thank you for using a complete sentence!

Reply:
I didn't get that one 😅

Reply:
![](https://media.giphy.com/media/h58E0JsuK3h3d8B1do/giphy.gif)

Reply:
@germa89 Ha! Most message strings that developers write are sentence fragments, and the Google developer documentation style guide (and basically all others) say that message strings should be full sentences--just like yours! I was trying to praise your work!

Reply:
Awww.... 

![](https://media.giphy.com/media/1zatzZDiWS73W/giphy-downsized-large.gif)